internal/cmd/object-storage/credentials/create/create.go

```diff
@@ -0,0 +1,128 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	objectStorageUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/objectstorage"
+)
+
+const (
+	expiresFlag          = "expires"
```

Comment:
This flag could be `expire-date` instead, to be aligned with the Portal and more clear. If it's just `expires` it is not obvious that it is expecting a date

![image](https://github.com/stackitcloud/stackit-cli/assets/56552880/89c09252-4fd8-4d17-9736-4d2f01f6e7c8)